src/components/RouteVideoPlayer.tsx

```diff
@@ -10,6 +10,7 @@ type RouteVideoPlayerProps = {
   class?: string
   routeName: string
   startTime: number
+  onEvent?: (event: 'load' | 'error') => void
```

Comment:
I think any props we expect to use in other parts of the app we shouldn't make optional

Just have optional props on reusable UI components like buttons, cards etc